tesseract_core/sdk/docker_cli_wrapper.py

```diff
@@ -0,0 +1,475 @@
+"""Docker for Tesseract usage."""
+
+import json
+import logging
+import os
+import shlex
+import subprocess
+from pathlib import Path
+
+logger = logging.getLogger("tesseract")
+
+
+class DockerWrapper:
```

Comment:
Thinking of a better name for this class. `CLIDockerClient`? To me it's more than a wrapper, and closely modeled after `DockerClient`, so the name should reflect that.